include/nighthawk/client/options.h

```diff
@@ -44,6 +45,10 @@ class Options {
   // Defaults to HTTP/1.1 if the user doesn't make an explicit selection.
   virtual Envoy::Http::Protocol protocol() const PURE;
 
+  // The following sets specific protocol options.
```

Comment:
nit: for http3

Reply:
Done in 1a55a9c07be191868cf9e318a4ff3e0a8da65d6d.